src/analysis/monotone-analyzer.h

```diff
@@ -143,6 +111,7 @@ struct MonotoneCFGAnalyzer {
   Lattice lattice;
   TransferFunction transferFunction;
   std::vector<BlockState<Lattice>> stateBlocks;
+  CFG* cfg = nullptr;
```

Comment:
Can we make the analyzer own the CFG?

Reply:
The CFG has been changed to a reference, but as we discussed offline, the analyzer should not own the CFG because we may want to do multiple analyses on the same CFG or possibly have the transfer function pre-process the CFG also.